view/breakpoint.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * SPDX-FileCopyrightText: 2022 Zeal 8-bit Computer <contact@zeal8bit.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * Before initializing the components/peripherals, create the callback set.
```

Comment:
This comment is a leftover that can be removed

Reply:
#removed